index.html

```diff
@@ -0,0 +1,145 @@
+     
+    </div>
+    <div class="headre-black-text">
+      <p>Gain the technical capabilities and the job-ready skills employers require. Through your ALX software engineering programme you will benefit from your peer community, unlock new experiences, and connect to opportunities that will make an impact.!</p>
+    </div>
+    <div class="container-fluid header-end">
+     <div class="header-end-block">
+      <h2>2022.10.15(THU) ~ 16(FRI)</h2>
+      <P>@ National Museum of Korea Art Center Nabi and more</P>
+     </div>
+    </div>
+  </div>
+<br>
+
+<div class="container-fluid main-program">
+  <section>
+    <div class="container center-block">
+      <h3>Main Program</h3>
+    <div class="red-line"></div>
+    <div class="row">
+      <div class="col-sm-2">
+        <img class="lectures-img" src="https://img.icons8.com/ios/50/FFFFFF/podium-with-speaker.png"/>
+          <h4>Lectures</h4>
+          <p>Listen to the speakers from various countries on various topics.</p>
+      </div>
+      <div class="col-sm-2"><img class="lectures-img" src="images/forum.png"/>
+        <h4>Forum</h4>
+        <p>Listen to the speakers from various countries on various topics.</p></div>
+      <div class="col-sm-2">
+        <img class="lectures-img" src="images/exha.png"/>
+        <h4>CC Exhibition</h4>
+        <p>Listen to the speakers from various countries on various topics.</p></div>
+      <div class="col-sm-2"><img class="lectures-img" src="https://img.icons8.com/ios/50/FFFFFF/podium-with-speaker.png"/>
+        <h4>Lectures</h4>
+        <p>Listen to the speakers from various countries on various topics.</p></div>
+      <div class="col-sm-2"><img class="lectures-img" src="images/ignite.png"/>
+        <h4>Lectures</h4>
+        <p>Listen to the speakers from various countries on various topics.</p></div>
+    </div>
+    </div> 
+  </section>
+  <div class="see-whole"><a class="see-the-whole" href="#">See The Whole Project</a></div>
+  <div class="button-cc"><button class="button-alx">Join ALX 2022-2023</button> </div>
+</div>
+
+<br>
+
+<div class="cotainer features">
+  <section>
+    <div class="feature-block">
+      <h3>Feature speakers</h3>
+      <div class="red-line"></div>
+      <div class="row feature-row">
+
+      </div>
+      <div class="Btn-text"><button id="textButton">Show More</button></div>
+    </div>
+
+    
+  </section>
+</div>
+
+<br>
+
+<div class="container-fluid partner-row">
+  <section>
+    <div class="partner-block">
+      <h3>Partner</h3>
+      <div class="red-line"></div>
+      <div class="row partner">
+        <div class="col-sm"><img src="images/mozilla.png" alt=""></div>
+        <div class="col-sm"><img src="images/google_logo.png" alt=""></div>
+        <div class="col-sm"><img src="images/naver.jpeg" alt=""></div>
+        <div class="col-sm"><img src="images/Airbnb.jpeg" alt=""></div>
+        
+        <div class="col-sm"><img src="images/Airbnb.jpeg" alt=""></div>
```

Comment:
- [ ] Please add an `alt` text that describes the image to all your images. This is so that screen readers can know what image it is and also so that the text will be displayed when the image fails to load.